modules/virt-configuring-cluster-eviction-strategy-cli.adoc

```diff
@@ -8,9 +8,6 @@
 
 You can configure an eviction strategy for a cluster by using the command line.
 
-:FeatureName: Configuring a cluster eviction strategy
-include::snippets/technology-preview.adoc[leveloffset=+1]
```

Comment:
Did this get some sort of announcement somewhere that it is not tech preview anymore? Just wondering if this might need change management.

Reply:
Sorry @aspauldi looks like we grabbed at the same time! 😆

Reply:
Hi @opayne1, Ha! We did!

We don't use change management in OpenShift Virtualization, so this is all set.

Reply:
Last I've checked, CNV doesn't use the change management process that core OCP does so it shouldn't need to go through that process. I did however add it to the RN for this release already in [this PR](https://github.com/openshift/openshift-docs/pull/82880).